app/src/tests/components/contents/metadata/Metadata.test.tsx

```diff
@@ -34,6 +34,17 @@ describe("Metadata component", () => {
     expect(screen.getByText(/branch/i)).toBeInTheDocument();
     expect(screen.getByText(/commit/i)).toBeInTheDocument();
     expect(screen.getByText(/remotes/i)).toBeInTheDocument();
+
+    act(() => screen.getByText(/platform/i).click());
```

Comment:
should never need to use act..... for user events use userEvent ... should be able to do `await userEvent.click` here

Reply:
Why do you prefer userEvent over act?

Reply:
I think userEvent is synchronous... we're not calling it with await anywhere else.

Reply:
sorry i forgot yeah dont need await....  act was around before await waitFor etc.. they all use act under the hood... Also uservent more closely resembles a user action.. which is our goal for testing frontend code.(it actually uses fireEvent under the hood)

Reply:
> uservent more closely resembles a user action

Ah cool, that's a good reason.